iquip/apps/builder.py

```diff
@@ -185,6 +185,49 @@ def value(self) -> str:
         return self.lineEdit.text()
 
 
+class _DateTimeEntry(_BaseEntry):
+    """Entry class for a date and time value.
+    
+    Attributes:
+        checkBox: The checkbox for the availability of the dateTimeEdit.
+        dateTimeEdit: The dateTimeEdit for the date and time value.
+    """
+
+    def __init__(self, name: str, parent: Optional[QWidget] = None):
+        """Extended."""
+        super().__init__(name, {}, parent=parent)
+        # widgets
+        self.checkBox = QCheckBox(self)
+        self.checkBox.setChecked(False)
+        self.checkBox.stateChanged.connect(self.updateDateTimeEditState)
+        currentDateTime = QDateTime.currentDateTime()
+        self.dateTimeEdit = QDateTimeEdit(currentDateTime, self)
+        self.dateTimeEdit.setCalendarPopup(True)
+        self.dateTimeEdit.setDisplayFormat("yyyy-MM-dd HH:mm:ss")
+        self.dateTimeEdit.setEnabled(False)
+        self.dateTimeEdit.setMinimumDateTime(currentDateTime)
+        # layout
+        self.layout.addWidget(self.checkBox)
+        self.layout.addWidget(self.dateTimeEdit)
+
+    @pyqtSlot()
+    def updateDateTimeEditState(self):
+        """Enables or disables the dateTimeEdit in according to the state of the checkBox.
+        
+        Once the state of the checkBox is changed, this is called.
+        """
+        self.dateTimeEdit.setEnabled(self.checkBox.isChecked())
+
+    def value(self) -> Optional[str]:
+        """Overridden.
+        
+        Returns the value of the dateTimeEdit in ISO format if it is enabled.
+        Otherwise returns None.
+        """
+        return self.dateTimeEdit.dateTime().toString(Qt.ISODate) if self.dateTimeEdit.isEnabled() \
```

Comment:
Please avoid using the backslash.

Reply:
Then, do you mean using a parentheses?
I think we should make an explicit line continuation using a backslash or a parentheses.

Reply:
If the line becomes too long, you have to separate the code into multiple lines.
For example, you should not use the if-else _expression_ (or, the ternary operator) in that case. You should rather use the if-else _statement_.

Backslashes are not that readable (although this case was quite simple) in general.
If you have to write a long line, parentheses are preferred as stated in [PEP8](https://peps.python.org/pep-0008/#maximum-line-length).